content/kotsadm/installing/system-requirements.md

```diff
@@ -48,6 +48,12 @@ IP addresses for these services can be found in [replicatedhq/ips](https://githu
 
 No outbound internet access is required for airgapped installations.
 
+## Airgapped Installs
+
+When application airgap bundle is uploaded through the Admin Console's UI, application images will be extracted and pushed to the registry from inside the Admin Console's pod.
+This requires extra ephemeral storage available on the node.
+The recommended amount of storage should be no less then the total size of application's images.
```

Comment:
```suggestion
The recommended amount of storage should be no less than the total size of application's images.
```

it's all I'm good for anymore 😂

Reply:
omg

Reply:
Is this accurate? Isn't the ephemeral storage requirement actually the size of the Airgap bundle archive in addition to the size of the applications's images uncompressed? I'm curious to understand the details here.